sdk/shared/src/main/scala/org/ergoplatform/sdk/multisig/Signer.scala

```diff
@@ -0,0 +1,17 @@
+package org.ergoplatform.sdk.multisig
+
+import org.ergoplatform.P2PKAddress
+import org.ergoplatform.sdk.SigmaProver
+
+case class Signer(prover: SigmaProver) {
```

Comment:
The file should be removed from this PR I suppose

Reply:
yes, removed